lib/pre2017.js

```diff
@@ -0,0 +1,97 @@
+const fetch = require('node-fetch')
+const cheerio = require('cheerio')
+const https = require('https')
+
+const PRE_URL = [
+  'https://codein.withgoogle.com/archive/2016',
+  'https://codein.withgoogle.com/archive/2015',
+  'https://www.google-melange.com/archive/gci/2014',
+  'https://www.google-melange.com/archive/gci/2013',
+  'https://www.google-melange.com/archive/gci/2012',
+  'https://www.google-melange.com/archive/gci/2011',
+  'https://www.google-melange.com/archive/gci/2010',
+]
+
+async function fetchText(url) {
+  const agent = new https.Agent({
+    rejectUnauthorized: false,
+  })
+  const res = await fetch(url, {
+    agent,
+  })
+  return await res.text()
+}
+
+async function fetchPre2017(url) {
+  const year = url.split('/').pop()
+  const responseHTML = await fetchText(url)
+  const html = cheerio.load(responseHTML)
+  let data = []
+  if (year == '2016' || year == '2015') {
+    const orgsList = html(`
+          section.home__winners
+          div.fixed-width
+          ul.no-style-list > li`)
+    data = orgsList.toArray().map(org => {
+      const orgHTML = cheerio.load(org)
+      return {
+        name: orgHTML('div.home__winner-box-header h3 a').text(),
+        winners: getUlList(
+          orgHTML,
+          'div.home__winner-box-content div:first-child ul li'
+        ),
+        finalists: getUlList(
+          orgHTML,
+          'div.home__winner-box-content div:last-child ul li'
+        ),
+      }
+    })
+  } else if (year == '2014' || year == '2013' || year == '2012') {
```

Comment:
Use `===` instead. Or, read [in operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) if you're interested.

Reply:
`in'  or `.includes()`?

Reply:
Whatever you feel better.